modules/installation-osp-default-kuryr-deployment.adoc

```diff
@@ -24,17 +24,32 @@ Use the following quota to satisfy a default cluster's minimum requirements:
 |Networks                | 250 - 1 needed per Namespace/Project
 |RAM                     | 112 GB
 |vCPUs                   | 28
-|Volume storage          | 175 GB
+|Volume storage          | 275 GB
 |Instances               | 7
 |Security groups         | 250 - 1 needed per Service and per NetworkPolicy
 |Security group rules    | 1000
-|Swift containers        | 2
-|Swift objects           | 1
-|Swift available space   | 10 MB or more
 |Load balancers          | 100 - 1 needed per Service
 |Load balancer listeners | 500 - 1 needed per Service-exposed port
 |Load balancer pools     | 500 - 1 needed per Service-exposed port
-|================================
+|==============================================================================================
+
+[IMPORTANT]
+====
+If OpenStack Object Storage (Swift) is available, it is used as the default backend for the {product-title} image registry. In this case, the volume storage requirement is 175GB.
+
+.Recommend Swift resources
```

Comment:
Feel free to ignore this table in review. It will be wiped out by the Swiftless installation changes.